deploy/charts/venafi-kubernetes-agent/README.md

```diff
@@ -0,0 +1,162 @@
+
+For chart installation, run the following to set a registry configuration
+file, so `helm` can authenticate to our private OCI registry:
+
+```shell
+export VENAFI_DOCKER_CONFIG_PATH="$(pwd)"
+export VENAFI_DOCKER_CONFIG_FILE="${TLSPK_DOCKER_CONFIG_PATH}/config.json"
+jsctl registry auth output --format=dockerconfig > "${VENAFI_DOCKER_CONFIG_FILE}"
+```
+
+To validate you registry credentials are working with `helm`, we can use it to
+show us the full list of values available to configure the chart:
+
+```shell
+export VENAFI_REGISTRY="eu.gcr.io/jetstack-secure-enterprise"
+helm show values oci://${VENAFI_REGISTRY}/charts/venafi-kubernetes-agent \
+  --registry-config "${VENAFI_DOCKER_CONFIG_FILE}"
+```
+
+**Note**: Feel free to alter the registry to the US equivalent if that is closerto you.
+For example: `export VENAFI_REGISTRY="us.gcr.io/jetstack-secure-enterprise"`
+
+### 2) Creating Venafi Service Account:
+
+First we need to create an OpenSSL key pair and save the private key securely.
+The private key is used by the agent and you should have a unique key for each agent you connect to the Venafi Control Plane.
+The public key will be added to the Venafi Control Plane as the service account credential and assigned to the appropriate team for ownership.
+
+```shell
+export VENAFI_NAMESPACE="venafi" VENAFI_SERVICE_ACCOUNT="example-cluster"
+openssl genrsa -out ${VENAFI_SERVICE_ACCOUNT}.pem
+openssl rsa -in ${VENAFI_SERVICE_ACCOUNT}.pem -pubout --out ${VENAFI_SERVICE_ACCOUNT}.pub
+```
+
+Now that you have both the private and public key we now need to use the Venafi Control Plane to create a service account.
+
+- Navigate to the [service accounts page](https://ui.venafi.cloud/service-accounts/) and select "New"
+- Add a unique name matching the variable we used, eg: "example-cluster"
+- Assign a team that owns this credential
+- The scope should be "Kuberentes Discovery" only.
+- Set the validity period of your pubic key up to a maximum of 365 days.
+- Now paste in the **public key** from the pair you generated.
+
+Once created, you will be returned to the service accounts list.
+Find your newest entry matching the name you entered, and copy the "Client ID" column.
+
+### 3) Deploying the chart:
+
+Now we have the service account, let us prepare a namespace with the relevant private key needed at runtime.
+
+```shell
+export VENAFI_CLIENT_ID="<PASTE YOURS HERE>"
+kubectl create namespace ${VENAFI_NAMESPACE}
+kubectl create secret generic agent-credentials -n ${VENAFI_NAMESPACE} \
+  --from-file=privatekey.pem=${VENAFI_SERVICE_ACCOUNT}.pem
+```
+
+Install the chart by setting the `config.clientId` field:
+
+```shell
+helm upgrade --install agent deploy/charts/venafi-kubernetes-agent \
+  --namespace ${VENAFI_NAMESPACE} \
+  --set config.clientId="${VENAFI_CLIENT_ID}"
+```
+
+### 4) Add Cluster in Venafi Control Plane
+
+- Go to "Installations" -> "Kuberentes Clusters" [here](https://ui.venafi.cloud/clusters-inventory) and click "Connect"
```

Comment:
This is really outside the scope of this PR but I find that we have to do these steps really bad UX.. why the hell do we need to pre-register clusters in VaaS?, this will defeat the point of having a helm chart and automation to deploy this at scale. 

We should raise a productboard issue on this one i think.. unless i am mistaken

Reply:
I agree it's not great. I haven't had a chance to raise it with product team yet. But as you say, out of the scope of this PR.

My intention is to have something that works `v0.1.0` and iterate from there before officially announcing.